test/e2e/e2e_test.go

```diff
@@ -902,6 +921,16 @@ func TestE2E(t *testing.T) {
 		checkMetricsCollection(t, promClient, "openshift-service-ca-operator")
 	})
 
+	// Test that signer CA metrics are collected
+	// Test that the operator's metrics endpoint is being read by prometheus
+	t.Run("signer-ca-metrics", func(t *testing.T) {
+		promClient, err := newPrometheusClientForConfig(adminConfig)
```

Comment:
Given initialization of a prometheus client involves a bunch of api calls, could the 2 metrics tests share its setup?  Maybe by nesting the tests e.g.

```golang
t.Run("metrics", func(t *testing.T) {
  promClient, err := ...
  ...
  t.Run("collection", func(t *testing.T) {
    ...
  }
  t.Run("signer-ca-expiry", func(t *testing.T {
    ...
  }
}
```

Reply:
now the tests run in parallel and share the setup!